app/controllers/products_controller.rb

```diff
@@ -1,5 +1,11 @@
 class ProductsController < ApplicationController
   def index
-    @pagy, @products = pagy Product.all.newest, items: Settings.number.digit_12
+    products = if params[:term].present?
+                 Product.search(params[:term])
+               else
+                 Product
+               end.newest
```

Comment:
```suggestion
    products = Product.search(params[:term]).newest
```
check param search nên để ở controller nhé